src/server/ua_server_internal.h

```diff
@@ -66,13 +65,13 @@ typedef enum {
 } UA_DiagnosticEvent;
 
 typedef struct channel_entry {
-    UA_TimerEntry cleanupCallback;
+    UA_DelayedCallback cleanupCallback;
```

Comment:
All the UA_TimerEntry callbacks are exchanged with UA_DelayedCallbacks of the eventloop